### ISSUE TITLE
OTA not asking for password in browser

### ISSUE THREAD

**Issue by R0CKB0TT0M** [stnkl/ESPEssentials#5]
First of all, thank you for this awesome library I love it.
Sadly, I have run into an issue using it on an esp8266 (esp01).
I tried the basic example with OTA Password and everything works great but when I go to HTTP://<myip>/update it never asks for a password it just lets me update without it.
Using ArduinoIDE 2.0.0-rc6 to push the OTA I get asked for the password.
From Experience, 99% of the Problems I have are just me being stupid but I really don't know what is going wrong here.

My Code:
```
#include <ESPEssentials.h>
void setup() {
  ESPEssentials::init("myproject","mypw");

}

void loop() {
  ESPEssentials::handle();
 Serial.println("it works");
}

```

**Comment by R0CKB0TT0M** [comment]
I've had a good night's sleep and after getting up I decided to take a look at the code. From what I can see It looks like the web interface was never meant to be protected. Is this intentional? It feels weird to have a password for direct uploads through the ide but then not bother when you upload a bin through the web interface.

**Comment by srwi** [comment]
Hi @R0CKB0TT0M, thank you for opening the issue. You are correct, this is an oversight. I will take care of it!

**Comment by srwi** [comment]
I implemented a fix in 2.1.1. It will be available in the Arduino IDE once their repository gets rescanned (probably within 24h). Thanks again for the bug report!
